articles/Getting-started-with-windows-forms-using-c-#/index.md

```diff
@@ -0,0 +1,197 @@
+**Windows forms** allow the programmers to create a graphical user interface (GUI) that allows the user to interact with the program visually. According to [Microsoft](https://docs.microsoft.com/en-us/dotnet/desktop/winforms/overview/?view=netdesktop-5.0), It provides one of the most productive ways to create desktop apps based on the visual designer provided in Visual Studio. Functionality like the drag-and-drop placement of controls makes it easy for the programmers to build the GUI.
+
+With *windows forms*, you can make basic applications like a calculator to high-level applications such as a library database management application that requires a connection to the database from the application.
+
+### Prerequisites
+1. A basic understanding of C# programming language.
+2. Visual studio installed on your system.
+
+If you don't have Visual Studio installed on your computer, you can check this article on how to set up the C# environment in Visual Studio [here](https://www.geeksforgeeks.org/setting-environment-c-sharp/).
+
+### Make a new project
+In this tutorial, we will make a simple calculator. Make a new project by opening the Visual Studio, then click on File -> New -> Project -> Visual C# -> Windows desktop and choose Windows Forms App (.NET Framework).
+
+### ToolBox
+The *Toolbox* window provides you with all the controls that will help you build the form. To display the Toolbox click on View -> Toolbox, or you can press Ctrl+Alt+X. To use one of the controls double click on one of them or simply drag and drop it.
+
+### Properties window
+The *Properties* window is used to edit the properties of the form or the control (button, textbox, etc.) that you click on. To open the properties window, you can click on View -> Properties window or you can simply click F4.   
+
+![properties window and toolbox](properties window and toolbox.png)
```

Comment:
This image is not loading because the filename has spaces. Can you please rename the file and update it here?

Reply:
We can move to the final review if this is fixed.

Reply:
@zolomohan Done.